api/v1alpha1/oadp_types.go

```diff
@@ -318,7 +311,9 @@ type VolumeOptions struct {
 
 // Features defines the configuration for the DPA to enable the tech preview features
 type Features struct {
-	// Contains data mover specific configurations
+	// (do not use warning) Contains data mover specific configurations
```

Comment:
here is not a deprecating, because user can longer use the field, so no need to use something like this, right? 
https://github.com/openshift/oadp-operator/blob/master/controllers/nodeagent.go#L94-L99

Reply:
@mateusoliveira43 Why are we not removing the `DataMover` configuration ?

Reply:
Because then user upgrading directly would not complete the upgrade (old DPA would not be valid and upgrade gets stuck)